Header header
bool isStopped
vector3D linear
vector3D angular